features/request_specs/engine_named_routes.feature

```diff
@@ -0,0 +1,83 @@
+Feature: engine named routes
+
+  Generated requeset specs inside a Rails Engine is able to use route and url helpers.
```

Comment:
This has a typo, but I'd merge this into the main request specs feature more like 
```suggestion
  Request specs inside a Rails Engine can include the engines routes to use url helpers.
```